src/test/consensus/Validations_test.cpp

```diff
@@ -707,21 +707,45 @@ class Validations_test : public beast::unit_test::suite
         TestHarness harness(h.oracle);
         Node a = harness.makeNode();
 
+        // simple cases
         Ledger ledgerA = h["a"];
         BEAST_EXPECT(ValStatus::current == harness.add(a.validate(ledgerA)));
-        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerA.id()));
+        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerA.id()) == 1);
+        harness.vals().expire();
+        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerA.id()) == 1);
+        harness.clock().advance(harness.parms().validationSET_EXPIRES);
+        harness.vals().expire();
+        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerA.id()) == 0);
 
-        // Keep the validation from expire
+        // use setSeqToKeep to keep the validation from expire
+        Ledger ledgerB = h["ab"];
+        BEAST_EXPECT(ValStatus::current == harness.add(a.validate(ledgerB)));
+        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerB.id()) == 1);
+        harness.vals().setSeqToKeep(ledgerB.seq(), ++ledgerB.seq());
         harness.clock().advance(harness.parms().validationSET_EXPIRES);
-        harness.vals().setSeqToKeep(ledgerA.seq());
         harness.vals().expire();
-        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerA.id()));
+        BEAST_EXPECT(harness.vals().numTrustedForLedger(ledgerB.id()) == 1);
+        // change toKeep
+        harness.vals().setSeqToKeep(++ledgerB.seq(), ++(++ledgerB.seq()));
```

Comment:
Similar to above, I'd rather see this as `setSeqToKeep(ledgerB.seq() + Ledger::Seq(1), ledgerB.seq() + Ledger::Seq(2))`

I'm realizing that the explicit `Ledger::Seq` can get unwieldy, so how about defining
```
constexpr Ledger::Seq one(1);
constexpr Ledger::Seq two(2);
```
Then you can do `ledgerB.seq() + one`, etc.

Reply:
Good idea, changed, thanks.